src/input_mocking.rs

```diff
@@ -246,9 +246,9 @@ impl MockInput for MutableInputStreams<'_> {
     }
 
     fn reset_inputs(&mut self) {
-        // WARNING: this *must* be updated when MutableInputStreams's fields change
+        // WARNING: this *must* be updated when MutableInputStreams' field changed

```

Comment:
```suggestion
        // WARNING: this *must* be updated when MutableInputStreams''s fields change
```
This suggestion was wrong.